recipes/corona_lineage_dynamics/meta.yaml

```diff
@@ -0,0 +1,92 @@
+{% set version = "0.1.5" %}
+
+package:
+  name: corona_lineage_dynamics
+  version: {{ version }}
+
+source:
+  url: https://github.com/hzi-bifo/corona_lineage_dynamics/releases/download/v0.1.5/corona_lineage_dynamics-v0.1.5.tgz
+  sha256: c8afa72a1645036b69772bb4d2a2e02372f1f5c72fe46e1ba6d215a26c805f5f
+
+build:
+  number: 0
+  run_exports:
+    - corona_lineage_dynamics >=0.1.2,<0.2
+
+requirements:
+    build:
+        - {{ compiler('c') }} 
+        - {{ compiler('cxx') }} 
+        - ta-lib # ['osx']
+        - make 
+        - boost-cpp 
```

Comment:
This should be a `host` dependency.
Consider using the new `libboost-devel` instead too!

Reply:
Added into host and run.